staramr/subcommand/Search.py

```diff
@@ -313,6 +317,15 @@ def run(self, args):
         else:
             logger.info("No --pointfinder-organism specified. Will not search the PointFinder databases")
             pointfinder_database = None
+        
+        if (args.plasmidfinder_database_type):
+            if args.plasmidfinder_database_type not in PlasmidfinderBlastDatabase.get_available_databases():
+                raise CommandParseException("The only Plasmidfinder bacteria(s) currently supported are " + str(
```

Comment:
Could you change this from **Plasmidfinder bacteria(s)** to **Plasmidfinder databases**.

Reply:
Yeah, fixed this in commit 65ad9c9